contracts/KyberDxMarketMaker.sol

```diff
@@ -479,6 +479,41 @@ contract KyberDxMarketMaker is Withdrawable {
         }
     }
 
+    event Execution(
+        bool success,
+        address caller,
+        address destination,
+        uint value,
+        bytes data
+    );
+
+    // FFU
+    function executeTransaction(
+        address destination,
+        uint value,
+        bytes data
+    )
+        public
+        onlyAdmin
+    {
+        if (destination.call.value(value)(data)) {
```

Comment:
I also told it to ilan, just putting require here is nicer. but nvm.

Reply:
Discussed f2f, will consider this change for future version.